### ISSUE TITLE
Model names are plural, leads to error

### ISSUE THREAD

**Issue by wymedia** [thedevdojo/voyager#355]
- Laravel Version: 5.3.26
- Voyager Version: 0.10.0
- PHP Version:
- Database Driver & Version:

### Description:
When creating a database table, the 'create model' button will create a model using plural (e.g. App\Videos). When one then uses the bread feature, it pre-fills the 'Model Name' with singular version (e.g App\Video). This results in an error. It can be solved by going into your code and change the model to the singular version manually.

Better would be that the 'create model' is immediately creating a singular version model (e.g. App\video)

**Comment by marktopper** [comment]
The problem is only with the database table being incorrect right?

Cause in this case I'm gonna make the system add the ´table´ property to the model to ensure that the correct table is being used.

**Comment by wymedia** [comment]
Well, not really.

If your table name is videos, I would assume the model is App\Video and not App\Videos. Just to be consistent with a table users and a model App\User. While the table is often plural, the model is often singular.

In the current version of Voyager, when the table name is videos, a model App\Videos is created (instead of App\Video). The problem is then that in the bread function the App\Video is proposed to the user. This leads to an error obviously. 

So not really a bug, but I think it should be changed for consistency with the rest of Laravel. The only thing that needs to happen is that the model should be singular.

**Comment by marktopper** [comment]
Okay. I will soon come up with a fix for this and let you know.

**Comment by victorpierredev** [comment]
Just to add to this issue, since the one I've opened was mark as duplicated. What I was experiencing what not a pluralization issue, but a casing issue. With composite words, the model creation uses snake case, while everywhere else uses camel case.

**Comment by marktopper** [comment]
What was the table-name you created @vpakg?

**Comment by victorpierredev** [comment]
Hey @marktopper , the table created was `uploaded_files`. It generated a model named Uploaded_files but the auto-filled information shows by default `App\UploadedFile`.

For a more detailed explanation, please view #386 

Thanks!

**Comment by BCleverly** [comment]
Just been looking into this as I've encountered this as well.

I noticed [on this line](https://github.com/the-control-group/voyager/blob/master/src/Commands/MakeModelCommand.php#L57) that you're taking the name straight from the form the user submits. If people use a plural e.g. teams/products, the model created is Teams/Products respectively, when it's the singular we need.

Laravel has a helper function for `str_singluar` and `str_plural`, all depends on how you want to approach this.
